### PR TITLE
added templates

### PR DESCRIPTION
**Pull request by rutujaingole**
wrt issue no. #3. Added the Issue Templates and PR template.

### CODE REVIEW

File: website/.github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Pull Request Template
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```
Comment (sarthakkundra) [inline website/.github/PULL_REQUEST_TEMPLATE.md:5]: Remove `which issue is fixed`

File: website/.github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Additional Context (Please include any Screenshots/gifs if relevant)
+
+...
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (sarthakkundra) [inline website/.github/PULL_REQUEST_TEMPLATE.md:30]: remove

File: website/.github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
```
Comment (sarthakkundra) [inline website/.github/PULL_REQUEST_TEMPLATE.md:32]: remove

File: website/.github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] I have added tests that prove my fix is effective or that my feature works
```
Comment (sarthakkundra) [inline website/.github/PULL_REQUEST_TEMPLATE.md:33]: Change to `I have added corresponding tests`.

File: website/.github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] New and existing unit tests pass locally with my changes
```
Comment (sarthakkundra) [inline website/.github/PULL_REQUEST_TEMPLATE.md:34]: remove

File: website/.github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] I have checked my code and corrected any misspellings
```
Comment (sarthakkundra) [inline website/.github/PULL_REQUEST_TEMPLATE.md:36]: remove

File: website/.github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] I have tagged the reviewers in a comment below incase my pull request is ready for a review
```
Comment (sarthakkundra) [inline website/.github/PULL_REQUEST_TEMPLATE.md:37]: remove

File: website/.github/ISSUE_TEMPLATE/user-feedback.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (sarthakkundra) [inline website/.github/ISSUE_TEMPLATE/user-feedback.md:1]: What will be the use case of this? I can see this potentially overlapping with the feature request or bug report

File: website/.github/ISSUE_TEMPLATE/science-contribution-and-review-.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (sarthakkundra) [inline website/.github/ISSUE_TEMPLATE/science-contribution-and-review-.md:1]: What is the use case of this?

File: website/.github/ISSUE_TEMPLATE/communication-suggestion.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (sarthakkundra) [inline website/.github/ISSUE_TEMPLATE/communication-suggestion.md:1]: Can see this potentially overlapping with feature request or bug report so remove this.

**Review by sarthakkundra — CHANGES_REQUESTED** [review]

**Comment by rutujaingole** [comment]
@sarthakkundra made all the requested changes. Let me know if any further changes are required!

**Comment by HarshCasper** [comment]
Hi @rutujaingole 

Can you fix the DCO? 

You can do the same by running: 

```
$ git rebase HEAD~11 --signoff
$ git push --force-with-lease origin
```

If possible please squash your commits as well using rebase operation.

**Comment by rutujaingole** [comment]
@sarthakkundra  I've made the changes. Please review them and let me know if any further changes are to be made!

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**